src/components/lockPage.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react'
+import { FiLock, FiUnlock} from 'react-icons/fi'
+import { FaBatteryHalf, FaBatteryQuarter, FaBatteryThreeQuarters, FaBatteryFull} from 'react-icons/fa'
```

Comment:
Om man ska ha en ikon för att visa batteriet är slut så måste vi importera den också.

```suggestion
import { FaBatteryEmpty, FaBatteryHalf, FaBatteryQuarter, FaBatteryThreeQuarters, FaBatteryFull} from 'react-icons/fa'
```

Reply:
Vet inte riktigt om vi kan se att den har 0 genom API:t, men jag slänger in en check för det